testapp/testdata.json

```diff
@@ -421,6 +432,468 @@
     "approved_go_live_at": null
   }
 },
+{
+  "model": "wagtailcore.pagerevision",
```

Comment:
There is a lot of pagerevision data here. Can you clean it up so that there is only one new pagerevision per edited page?
The easiest way probably to edit the json by hand.